src/components/output/CopyBox.js

```diff
@@ -1,18 +1,28 @@
 import { CopyIcon } from "@chakra-ui/icons";
-import { Box, Button, Tooltip, useColorModeValue } from "@chakra-ui/react";
+import {
+  Box,
+  Button,
+  Tooltip,
+  Editable,
+  EditableTextarea,
+  EditablePreview,
+  useColorModeValue,
+} from "@chakra-ui/react";
 
 import { useState } from "react";
 
-export default function CopyBox({ children }) {
+export default function CopyBox({ children, editable = false }) {
```

Comment:
Clicking the "copy" icon doesn't copy the updated text. Will probably need to use useState / useRef

Reply:
Didn't realise this problem - Fixed.